sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -146,10 +151,10 @@ public virtual async Task<Response<TableItem>> CreateAsync(CancellationToken can
         /// <param name="entity">The entity to insert.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         /// <returns>The inserted Table entity.</returns>
-        public virtual Response<ReadOnlyDictionary<string, object>> Insert(IDictionary<string, object> entity, CancellationToken cancellationToken = default)
+        public virtual Response<ReadOnlyDictionary<string, object>> CreateEntity(IDictionary<string, object> entity, CancellationToken cancellationToken = default)
```

Comment:
could you remind me what happens if the user calls `CreateEntity` and the entity already exists?

Reply:
It throws an exception from the service!

Reply:
It will be good if we add this to the docstring of the method, so that the user knows.